### PR TITLE
Add andres zarta to maintainers.json

### PR DESCRIPTION
**Pull request by andres-zartab**

### CODE REVIEW

File: config/maintainers.json [hunk 1]
```diff
@@ -130,6 +130,16 @@
...
+      "show_on_website": true,
+      "name": "Andrés Zarta",
+      "link_text": null,
+      "link_url": null,
+      "avatar_url": null,
+      "bio": "Data Scientist with Operations Research background. Started learning Go to improve my software development skills and ended up loving it for its simplicity and amazing community."
     }
   ]
-}
+}
```
Comment (tehsphinx) [inline config/maintainers.json]: Can you undo this change?

Comment (tehsphinx) [reply config/maintainers.json]:
```suggestion
}

```

Comment (tehsphinx) [reply config/maintainers.json]: Note: files committed to git optimally have an empty line at the end

File: config/maintainers.json [hunk 2]
```diff
@@ -130,6 +130,16 @@
       "link_url": null,
       "avatar_url": null,
       "bio": null
+    },
+    {
+      "github_username": "andres-zartab",
+      "alumnus": true,
```
Comment (tehsphinx) [inline config/maintainers.json]: This would mean you were a maintainer and have stopped being one.

```suggestion
      "alumnus": false,
```

File: config/maintainers.json (hunk 1 above)
Comment (andres-zartab) [reply config/maintainers.json]: Will fix it now!

File: config/maintainers.json (hunk 2 above)
Comment (andres-zartab) [reply config/maintainers.json]: Oh, I didn't got that right!

File: config/maintainers.json [hunk 3]
```diff
@@ -130,6 +130,16 @@
...
+      "show_on_website": true,
```
Comment (tehsphinx) [inline config/maintainers.json]:
```suggestion
      "show_on_website": false,
```

Set this to false for now. It will later be updated once new maintainers grow into more experienced maintainers.

File: config/maintainers.json (hunk 1 above)
Comment (andres-zartab) [reply config/maintainers.json]: Is it good enough to just accept your commit suggestion?

File: config/maintainers.json (hunk 2 above)
Comment (tehsphinx) [reply config/maintainers.json]: No problem. I did the same 😁

File: config/maintainers.json (hunk 1 above)
Comment (tehsphinx) [reply config/maintainers.json]: yes you can do that

**Comment by andres-zartab** [comment]
Thanks for your review and suggestions! :)
